pkg/systemd/host.js

```diff
@@ -22,6 +22,7 @@ define([
     "base1/cockpit",
     "base1/mustache",
     "domain/operation",
+    "performance/dialog",
```

Comment:
No, the directory and module have different names. That's the way it is with the realmd stuff too.